internal/pkg/cli/validate_test.go

```diff
@@ -147,6 +147,63 @@ func TestValidateEnvironmentName(t *testing.T) {
 	}
 }
 
+func TestValidatePipelineName(t *testing.T) {
+	testCases := map[string]struct {
+		val     interface{}
+		appName string
+
+		wanted            error
+		wantedErrorSuffix string
+	}{
+		"string as input": {
+			val:    "hello",
+			wanted: nil,
+		},
+		"number as input": {
+			val:    1234,
+			wanted: errValueNotAString,
+		},
+		"string with invalid characters": {
+			val:    "myPipe!",
+			wanted: errValueBadFormat,
+		},
+		"longer than 100 characters": {
+			val:               strings.Repeat("s", 101),
+			wantedErrorSuffix: fmt.Sprintf(fmtErrPipelineNameTooLong, 90),
+		},
+		"longer than 100 characters with pipeline-[app]": {
+			val:               strings.Repeat("x", 86),
```

Comment:
That is a much better way of making a long string than what I've done in the past! 😆

Reply:
i found it in someone else's test, i didn't think of it either 😂